test/integration/targets/win_acl_inheritance/tasks/main.yml

```diff
@@ -0,0 +1,173 @@
+---
+ # Test setup
+- name: remove test folder for baseline
+  win_file:
+    path: "{{test_win_acl_inheritance_path}}"
+    state: absent
+
+- name: create test folders
+  win_file:
+    path: "{{test_win_acl_inheritance_path}}\\folder"
```

Comment:
I thought you had to double quote a variable like "{{variable}}" if it was at the beginning unless that has changed recently?